### ISSUE TITLE
Too strict bounds on base

### ISSUE THREAD

**Issue by sternenseemann** [lspitzner/multistate#9]
Builds fine with `base` 4.15 at least, haven't tested 4.16, but might work?

**Comment by jneira** [comment]
There is a pr to allow ghc-9.0.1 and base-4.15: https://github.com/lspitzner/multistate/pull/8

**Comment by andys8** [comment]
Currently allows `base >= 4.11  && <4.17`. Issue can be closed.

https://github.com/lspitzner/multistate/blob/5179cb21787d488e6049ae6e853cb0c101fd04fd/multistate.cabal#L71

**Comment by sternenseemann** [comment]
True, although the `hspec` bound can be relaxed now  as well, hspec 2.9 works fine, but is prohibited.

**Comment by andys8** [comment]
[Jep, that would be great](https://github.com/lspitzner/multistate/pull/12#issuecomment-1367540772)

**Comment by Bodigrim** [comment]
As a Hackage trustee I made a revision: https://hackage.haskell.org/package/multistate-0.8.0.4/revisions/
